package.json

```diff
@@ -26,10 +26,14 @@
     "@apollo/federation": "file:federation-js",
     "@apollo/gateway": "file:gateway-js",
     "@apollo/query-planner-wasm": "file:query-planner-wasm",
+    "@apollo/harmonizer": "file:harmonizer",
     "@apollographql/apollo-tools": "0.4.9",
     "apollo-federation-integration-testsuite": "file:federation-integration-testsuite-js"
   },
   "devDependencies": {
+    "@rollup/plugin-commonjs": "^17.0.0",
+    "@rollup/plugin-node-resolve": "^11.0.1",
+    "@rollup/plugin-typescript": "^8.1.0",
```

Comment:
While this package was added to the `package.json` in the PR this implementation was extracted from in #305, it does not appear to be actually used in the `rollup.config.js`.

```suggestion
```